apps/devhub/tasks.py

```diff
@@ -64,30 +64,30 @@ def validate_and_submit(addon, file_, listed=None):
 
 @task
 @write
-def submit_file(addon_pk, file_pk):
+def submit_file(addon_pk, upload_pk):
     addon = Addon.unfiltered.get(pk=addon_pk)
-    file_ = FileUpload.objects.get(pk=file_pk)
-    if file_.passed_all_validations:
-        create_version_for_upload(addon, file_)
+    upload = FileUpload.objects.get(pk=upload_pk)
+    if upload.passed_all_validations:
+        create_version_for_upload(addon, upload)
     else:
-        log.info('Skipping version creation for {file_id} that failed '
-                 'validation'.format(file_id=file_pk))
+        log.info('Skipping version creation for {upload_pk} that failed '
+                 'validation'.format(upload_pk=upload_pk))
 
 
 @atomic
-def create_version_for_upload(addon, file_):
-    if (addon.fileupload_set.filter(created__gt=file_.created,
-                                    version=file_.version).exists()
-            or addon.versions.filter(version=file_.version).exists()):
-        log.info('Skipping Version creation for {file_id} that would cause '
-                 'duplicate version'.format(file_id=file_.pk))
+def create_version_for_upload(addon, upload):
+    if (addon.fileupload_set.filter(created__gt=upload.created,
+                                    version=upload.version).exists()
+            or addon.versions.filter(version=upload.version).exists()):
+        log.info('Skipping Version creation for {upload_pk} that would cause'
```

Comment:
Did this line get too long? I like to have trailing spaces instead of leading spaces.